plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityColor.java

```diff
@@ -354,4 +385,4 @@ public static void register() {
             return object.getAllowedColors();
         });
     }
-}
+}
```

Comment:
wot

Reply:
no that's you wonked the end-of-file

Reply:
how do I fix this? I'm not getting any errors, and there's no extra characters after this

Reply:
it's the *lack* of a newline character that's the issue